app/controllers/organizations/adoptable_pets_controller.rb

```diff
@@ -1,48 +1,61 @@
-class Organizations::AdoptablePetsController < Organizations::BaseController
-  include ::Pagy::Backend
-
-  skip_before_action :authenticate_user!
-  skip_verify_authorized only: %i[index]
-  before_action :set_likes, only: %i[index show],
-    if: -> { allowed_to?(:index?, Like) }
-  helper_method :get_animals
-
-  def index
-    @q = authorized_scope(Pet.unadopted, with: Organizations::AdoptablePetPolicy).ransack(params[:q])
-    @pagy, paginated_adoptable_pets = pagy(
-      @q.result.includes(:adopter_applications, :matches, images_attachments: :blob),
-      limit: 9
-    )
-    @pets = paginated_adoptable_pets
-  end
+module Organizations
+  class AdoptablePetsController < Organizations::BaseController
+    include ::Pagy::Backend
+
+    skip_before_action :authenticate_user!
+    skip_verify_authorized only: %i[index]
+    before_action :set_likes, only: %i[index show],
+      if: -> { allowed_to?(:index?, Like) }
+    helper_method :get_animals
```

Comment:
An aside comment - I feel like this method should be called `get_breeds` instead